consensus/state.go

```diff
@@ -648,10 +648,10 @@ func (cs *ConsensusState) handleMsg(mi msgInfo, rs RoundState) {
 	case *VoteMessage:
 		// attempt to add the vote and dupeout the validator if its a duplicate signature
 		// if the vote gives us a 2/3-any or 2/3-one, we transition
-		err := cs.tryAddVote(msg.Vote, peerKey)
-		if err == ErrAddingVote {
-			// TODO: punish peer
-		}
+		_ = cs.tryAddVote(msg.Vote, peerKey)
```

Comment:
dont add it to the normal `make install`. just a separate command that we can run at our leisure for now ...